sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/grouping.scala

```diff
@@ -79,6 +79,29 @@ case class Cube(groupByExprs: Seq[Expression]) extends GroupingSet {}
 // scalastyle:on line.size.limit line.contains.tab
 case class Rollup(groupByExprs: Seq[Expression]) extends GroupingSet {}
 
+// scalastyle:off line.size.limit line.contains.tab
+@ExpressionDescription(
+  usage = """
+    _FUNC_(([col1[, col2 ..]])*) - create a multi-dimensional grouping sets using the specified columns
```

Comment:
We need `ExpressionDescription` for these grouping analytics exprs? We already have documents for them though: https://spark.apache.org/docs/3.0.1/sql-ref-syntax-qry-select-groupby.html

Reply:
> We need `ExpressionDescription` for these grouping analytics exprs? We already have documents for them though: https://spark.apache.org/docs/3.0.1/sql-ref-syntax-qry-select-groupby.html

Don't need these after this pr.

Reply:
> doc need to be changed a lot too.

What does this mean?

Reply:
> > doc need to be changed a lot too.
> 
> What does this mean?

Means after this done, doc of https://spark.apache.org/docs/3.0.1/sql-ref-syntax-qry-select-groupby.html  need to change it's content.